### PR TITLE
auth-challenge - Jason

### PR DESCRIPTION
**Pull request by JasonKundi**

### CODE REVIEW

File: src/server/controllers/user.js [hunk 1]
```diff
@@ -8,28 +8,37 @@ const jwtSecret = 'mysecret';
...
-    const passwordsMatch = false;
+    const passwordsMatch = bcrypt.compareSync(password, foundUser.password);
 
     if (!passwordsMatch) {
         return res.status(401).json({ error: 'Invalid username or password.' });
     }
 
-    const token = null;
-
+    const token = jwt.sign(username, jwtSecret)
```
Comment (therearedoors) [inline src/server/controllers/user.js:41]: maybe await needed here for the jwt.sign call?

File: src/server/controllers/movie.js [hunk 2]
```diff
@@ -14,13 +14,22 @@ const createMovie = async (req, res) => {
     const { title, description, runtimeMins } = req.body;
 
     try {
-        const token = null;
+        const token = jwt.verify(token, jwtSecret)
```
Comment (alsyoufomar) [inline src/server/controllers/movie.js:17]: where did u get the token from?

File: src/server/controllers/user.js (hunk 1 above)
Comment (alsyoufomar) [reply src/server/controllers/user.js:41]: I don't think so

**Review by alsyoufomar — COMMENTED** [review]
overall your code is so good and neat(not repeating your code),  well done Jason!

**Pull request closed without merging**